contrib/seeds/generate-seeds.py

```diff
@@ -46,9 +46,7 @@ def name_to_bip155(addr):
     '''Convert address string to BIP155 (networkID, addr) tuple.'''
     if addr.endswith('.onion'):
         vchAddr = b32decode(addr[0:-6], True)
-        if len(vchAddr) == 10:
-            return (BIP155Network.TORV2, vchAddr)
```

Comment:
I think an error message specific for v2 not being supported would be better than the defualt error message

Reply:
I guess `makeseeds.py` is still producing these? It should probably ignore Torv2's from the DNS seeder completely.

Reply:
Thanks, updated `generate-seeds.py` to ignore torv2 rather than raise.